content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md

```diff
@@ -0,0 +1,150 @@
+
+![](/img/ws-image6.png)
+
+2﻿. Complete the setup.
+
+![](/img/ws-image7.png)
+
+Click Next and Select “Internal App” then Finish
+
+**Step 3:** **Export the SAML 2.0 IdP metadata**
+
+1. Click Next – Configure the Sign On settings
+
+   Two options available: **View Setup Instructions** which steps through the SAML config or **Identity Provider metadata** will produce an XML that can be loaded into Aruba Central.
+
+   Suggestion: Click **Identity Provider metadata** and save the XML data to a file.
+
+   ![](/img/ws-image9.png)
+2. C﻿lick Next
+3. Select Internal app and Click Finish
+
+**Step 4: Create SAML Authorization Profile in GreenLake Cloud Platform**
+
+1. Log into GreenLake and click Menu > Manage > Authentication and Click Set Up SAML Connection.
+
+   *Before you can add a new SAML configuration, you must have at least one user account with that domain already enabled in GreenLake. Also, you must be logged into GreenLake with an account from that domain in order to enable SSO for that domain.*
+
+   ![](/img/ws-image10.png)
+2. Type in the domain you want to enable SSO on:
+
+   ![](/img/ws-image11.png)
+3. Input the metadata from the above step
+
+   While GreenLake does support manually entering this info, it’s recommended to simply upload the XML metadata that was downloaded in the previous step. Select Metadata File and select the XML. Then click Next.
+
+   ![](/img/ws-image12.png)
+4. Enter the SAML attributes to match what was entered in Okta and set the idle timeout value as well.
+
+   ![](/img/ws-image13.png)
+5. Then click Next.
+6. Create a recovery user in the event SSO fails, an admin will still be able to access GreenLake.
+
+   ![](/img/ws-image14.png)
+
+   Congratulations SSO will now be enabled for GreenLake as well as the Aruba Central application. Log out and on the GreenLake home page, click Sign in with SSO.
+
+**Testing and Troubleshooting:**
+
+On the GreenLake Cloud Platform home page, Click Sign in with SSO.
+
+![](/img/ws-image15.png)
+
+![](/img/ws-image16.png)
+
+Enter the SSO credentials and you will be redirected to Okta to authenticate. Once you successfully authenticate, you will be redirected back to GreenLake. You can then click on the Aruba Central application and given access based on the configured role/permissions.
+
+**Additional Notes:**
+
+* There must be at least **one** verified user belonging to the **Domain** prior to configuration.
+* In order to configure SSO, you must be logged into GreenLake with a user from the domain.
+* SSO user access is determined by the “role_name” attribute included in the SAML hpe_ccs_attribute provided by the IdP.
+* SSO users can initiate a Single Sign On request by trying to log in to Aruba Central (SP-initiated login).
```

Comment:
Change "... log in to..." to "...log into..."

Reply:
Updated:
* SSO users can initiate a Single Sign On request by trying to log into Aruba Central (SP-initiated login).